Templates/CDS-CSharp/run.csx

```diff
@@ -0,0 +1,67 @@
+﻿#load ".\SecurityHelper.csx"
+#load ".\TraceWriterTelemetryBridge.csx"
+
+using Microsoft.CommonDataService;
+using Microsoft.CommonDataService.CommonEntitySets;
+using Microsoft.CommonDataService.Configuration;
+using Microsoft.CommonDataService.ServiceClient.Security;
+using Microsoft.CommonDataService.ServiceClient.Security.Credentials;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+class EnvironmentValues
+{
+    internal const string Tenant = "<REPLACE WITH YOUR TENANT INFORMATION>";
+    internal const string EnvironmentId = "<REPLACE WITH YOUR ENVEIRONMENT ID INFORMATION>";
+    internal const string ApplicationId = "<REPLACE WITH YOUR APPLICATION ID INFORMATION>";
+    internal const string ApplicationSecret = "<REPLACE WITH YOUR APPLICATION SECRET INFORMATION>";
+    internal const string RedirectUri = "<REPLACE WITH YOUR REDIRECT URI INFORMATION>";
+}
+
+public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, TraceWriter log)
+{
+    log.Info($"C# HTTP trigger function processed a request. RequestUri={req.RequestUri}");
+
+    if (req.Headers.Authorization == null)
+    {
+        return req.CreateResponse(HttpStatusCode.Forbidden);
+    }
+
+    using (var client = await SecurityHelper.GetClientFromConfiguration(req, log))
+    {
+        // Query product categories for Surface
+        // TODO: pick this from req parameters
+        var query = client.GetRelationalEntitySet<ProductCategory>()
+            .CreateQueryBuilder()
+            .Where(pc => pc.Name == "Surface")
```

Comment:
I had purposely removed it to just get the Surface from req parameters but noticed the rest of the code needs it. Adding it back. Will take care of the req params some time later